community/best_practices_managing_issues.md

```diff
@@ -0,0 +1,58 @@
+# Best practices for managing onnx issues
+
+## Enforce templates
+Currently there are no templates for creating issues in onnx. This makes it very difficult for community members to triage issues. Going forward templates shall be added and maintained for ONNX issues.
+
+Following templates shall be added
+* ONNX Bug
+* ONNX Question
+* Propose New Feature, Operator or Enhancement
+
+
+## Label issues
+
+* Going forward all issues must be labeled. 
+* Multiple labels should be used to properly capture the issue. For example when a user logs a shape inference bug for a particular operator.  [bug][shape-inference] labels should be added.
```

Comment:
So do we have "shape inference" in labels right now? I think it deserves. 
BTW, How about "checker"? I saw there are a few issues talking about checker not checking the errors.